server/test/support/TestQuestionBank.java

```diff
@@ -560,7 +560,8 @@ public QuestionModel maybeSave(
         question.loadQuestionDefinition();
       } catch (InvalidQuestionTypeException | UnsupportedQuestionTypeException e) {
         throw new IllegalArgumentException(
-            "Questions in the TestQuestionBank better be supported QuestionTypes.");
+            "Questions in the TestQuestionBank better be supported QuestionTypes. "
```

Comment:
(not-serious) This message isn't threatening enough. Can you add a knife emoji?

Reply:
😂